Cargo.toml

```diff
@@ -54,6 +54,8 @@ semver = { version = "1.0.14", default-features = false, features = ["serde", "s
 term = { version = "0.7.0", default-features = false }
 uuid = { version = "1.2.2", features = ["serde"] }
 
+nix-config-parser = { git = "https://github.com/DeterminateSystems/nix-config-parser" }
```

Comment:
We will need to prep this crate for publishing as this crate is published. I'm not in a huge rush to do this though! It's okay if this happens after this next release release since it will be focused on merging there.